scripts/040-python.sh

```diff
@@ -1,19 +1,24 @@
 #!/bin/bash
 set -eux -o pipefail
 
-# Package install
+user=$(id -un 1000)
+home=/home/$user
```

Comment:
Question: These appear unused?

Reply:
home is unused.
9f27446d064009d84451594018020a02c7b2c7d3